micrometer-core/src/main/java/io/micrometer/core/instrument/observation/DefaultMeterObservationHandler.java

```diff
@@ -42,17 +43,38 @@ public class DefaultMeterObservationHandler implements MeterObservationHandler<O
 
     private final MeterRegistry meterRegistry;
 
+    private final boolean shouldCreateLongTaskTimer;
+
+    /**
+     * Creates the handler with the default configuration.
+     * @param meterRegistry the MeterRegistry to use
+     */
     public DefaultMeterObservationHandler(MeterRegistry meterRegistry) {
         this.meterRegistry = meterRegistry;
+        this.shouldCreateLongTaskTimer = true;
+    }
+
+    /**
+     * Creates the handler with the defined Meters to use when the handler records the
+     * received events.
+     * @param meterRegistry the MeterRegistry to use
+     * @param metersToIgnore the Meters that should not be created when events are handled
```

Comment:
I would change "events" to "Observations" so people aren't confused with `Event`s that are part of Observations.